lib/pages/whiteboard/whiteboard_message_info_page.dart

```diff
@@ -36,70 +36,92 @@ class _WhiteboardMesssageInfoPageState
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(title: const Text('Message Info')),
-      body: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          const Text('Message text: '),
-          Form(
-            key: _formKey,
-            child: Row(
+      body: Padding(
```

Comment:
There really ought to be a GUI to do all this stuff for you, like the way JavaFX does it